docs/support/compatibility.md

```diff
@@ -14,9 +14,10 @@ The table below outlines the backward compatibility between Octopus Server and r
 | 3.5 ➜ 2018.2     | 3.3 ➜ 4.30.3                       | 3.5 ➜ latest    | 3.0 ➜ latest | 3.3 ➜ latest    |
 | 2018.2 ➜ 2018.12 | 4.30.7 ➜ 4.47.0                    | 3.5 ➜ latest    | 3.0 ➜ latest | 3.3 ➜ latest    |
 | 2019.1*           | 5.0.0 ➜ 5.2.7                      | 3.5 ➜ latest    | 4.0 ➜ latest | 5.0 ➜ latest    |
-| 2019.2* ➜ latest | 6.0.0 ➜ latest                     | 3.5 ➜ latest    | 4.0 ➜ latest | 5.0 ➜ latest    |
+| 2019.2* ➜ 2022.1 | 6.0.0 ➜ latest                     | 3.5 ➜ latest    | 4.0 ➜ latest | 5.0 ➜ latest    |
+| 2022.2** ➜ latest | 6.0.0 ➜ latest                     | 3.5 ➜ latest    | 6.2 ➜ latest | 5.0 ➜ latest    |
 
-## &ast; Partial forwards compatibility
+### &ast; Partial forwards compatibility
```

Comment:
I changed this to a smaller heading since I got a bit confused seeing the footnote be the same size as other headings. I thought it was a separate section entirely. I added the new footnote as a smaller heading as well so you can see the difference https://dev-a.octopus.com/docs/support/compatibility. But if we want to keep it this size I'll update the new footnote as well.

Reply:
Unfortunately, you can't make it an `H3` as the preceding header is an `H1`. DocSync as part of the build will fail the build as it has to have headings. That's the only reason I changed it back to an H2. I agree it doesn't make sense as a top-level heading since it's a footnote. I assume it was done historically because of the additional text (as opposed to just a single sentence) and the corresponding table. 

You could keep it as an H3 by adding a separate H2 at the very top of the page (but underneath the page yaml metadata) e.g.:

```
## Octopus Server compatibility
```

Reply:
Oh I didn't know that! That makes sense. I'll add that H2 and see how it looks